packages/ui/src/components/primitives/Message.tsx

```diff
@@ -0,0 +1,234 @@
+// This component file is based off - https://www.youtube.com/watch?v=vPRdY87_SH0
+import type {
```

Comment:
Make sure messages work in light & dark theme

Reply:
Done! 👍